sandbox/libs/authn/src/main/java/org/opensearch/authn/noop/NoopSubject.java

```diff
@@ -54,4 +54,10 @@ public String toString() {
     public void login(AuthenticationToken authenticationToken) {
         // Do nothing as noop subject is always logged in
     }
+
```

Comment:
Can you remind me of the purpose of the NoopSubject? I am not sure what we use it for.

Reply:
When identity becomes part of core, `NoopSubject` will provide a method for making the identity constructs pass-through. When identity is disabled, NoopSubject is the global subject and methods like `isPermitted()` will always return true. It's a way of disabling identity.

Reply:
Oh okay gotcha. Thank you